### PR TITLE
More matches and labels

### PR DESCRIPTION
**Pull request by FazanaJ**
Make clean also makes sure to kill the Linker now too, so you don't have to delete it yourself.

(Sorry it's another big one, just how documentation be sometime)

### CODE REVIEW

File: src/camera.c [hunk 1]
```diff
@@ -853,9 +880,133 @@ void func_80068508(s32 arg0) {
 }
 
 GLOBAL_ASM("asm/non_matchings/camera/func_80068514.s")
-GLOBAL_ASM("asm/non_matchings/camera/func_80068BF4.s")
+
+typedef struct unk80068BF4 {
```
Comment (Ryan-Myers) [inline src/camera.c:884]: Should the struct definition and GFX macros be moved elsewhere?

**Review by Ryan-Myers — APPROVED** [review]
LGTM. Nice work with the matches AND the documentation.

File: src/camera.c (hunk 1 above)
Comment (FazanaJ) [reply src/camera.c:884]: I shall fix that

Comment (FazanaJ) [reply src/camera.c]: I have fixed that.

**Review by DavidSM64 — APPROVED** [review]
LGTM

**Pull request merged**